### DISCUSSION TITLE
[Feature] Free up space button

### DISCUSSION THREAD

**Discussion by nett00n** [immich-app/immich#3546] [Feature Request]
### The feature

I'm sure we need a button to delete local photos, which are already uploaded to server, like in Google Photo mobile app.
I think that pressing this button must delete photos by these rules:
- Older than TTL configured in app (week by default?)
- Not liked
- Already uploaded to server

it is different feature than [Delete on mobile only](https://github.com/immich-app/immich/discussions/2190), but i think this feature needs `#2190` to be done before

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #165
